diff --git a/base-notebook/Dockerfile b/base-notebook/Dockerfile
index 9a3a836..c67c054 100644
--- a/base-notebook/Dockerfile
+++ b/base-notebook/Dockerfile
@@ -126,6 +126,7 @@ RUN set -x && \
         'notebook' \
         'jupyterhub' \
         'jupyterlab' && \
+    conda update -y -c conda-forge mamba && \
     rm micromamba && \
     # Pin major.minor version of python
     mamba list python | grep '^python ' | tr -s ' ' | cut -d ' ' -f 1,2 >> "${CONDA_DIR}/conda-meta/pinned" && \
